src/components/MarkdownEditor/index.tsx

```diff
@@ -4,6 +4,7 @@ import Editor, { IToolbar } from 'for-editor-dark';
 interface IMarkdownEditorProps {
   value: string;
   handleChange(value: string): void;
+  shouldBeChangelog: boolean;
```

Comment:
If you make `shouldBeChangelog` optional, you won't ever have to specify it as `false` on the components.

Reply:
Resolved by 8f0cdf5